resources/strings/strings.xml

```diff
@@ -76,6 +76,7 @@
 	<string id="Weather">Weather</string>
 	<string id="Humidity">Humidity</string>
 	<string id="Pressure">Pressure</string>
+	<string id="PulseOx">Pulse Ox</string>
```

Comment:
Please also add this string to the remaining 19 translation files, so that it can be easily translated in future.

Reply:
Done